packages/web-components/src/components/leaving-ibm/leaving-ibm.scss

```diff
@@ -30,19 +30,23 @@
 :host(#{$dds-prefix}-leaving-ibm-modal-body) {
   @extend :host(#{$prefix}-modal-body);
 
+  #{$prefix}-link {
+    @include carbon--type-style('body-long-02', true);
+  }
```

Comment:
I don't think you need to override it. The component has a setting `size="lg"`. Is there a reason we can't use that instead of overwriting the `CSS`?

Reply:
Also, it looks like `body-short-01` from the original component is getting the higher specificity.

Reply:
@photodow for some reason when I tested `size="lg"`, it didn't show any difference. I thought it was broken so I added the override. I just retested and it works. I updated the code. I guess my local build had a hiccup.